proposals/0000-add-support-for-css-color-function.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Add support for CSS color() function
+author:
+  - Ryan Linton
+  - Yulian Glukhenko
+date: 2023-11-20
+---
+
+# RFC0000: Add support for CSS color() function
+
+## Summary
+
+React Native does [not currently support](https://github.com/facebook/react-native/issues/41517) wide color gamut color spaces (i.e. display-p3). This proposal discusses adding support for the CSS color() function enabling support for additional wide-gamut color spaces.
+
+## Basic example
+
+```js
+StyleSheet.create({
+  backgroundColor: "color(display-p3 1 0.5 0)",
+  color: "color(display-p3 1 0.5 0 / .5)",
+});
+```
```

Comment:
Should we also handle inline styles?
```jsx
<View style={{backgroundColor:color(display-p3 1 0.5 0)}} />
```

I think we should, and it should be an easy extension. Thoughts?

Reply:
Oh definitely. I'll add it to the example.